selendroid-standalone/src/main/java/io/selendroid/standalone/server/model/SelendroidStandaloneDriver.java

```diff
@@ -164,8 +164,12 @@ public void addToAppsStore(File file) throws AndroidSdkException {
         // using "android" as the app name, because that is the desired capability default in
         // selenium for
         // DesiredCapabilities.ANDROID
+          File androidAPK = androidDriverAPKBuilder.extractAndroidDriverAPK();
+          if(serverConfiguration != null && serverConfiguration.isDeleteTmpFiles()) {
+              androidAPK.deleteOnExit(); //Deletes temporary files if flag set
```

Comment:
seems like something we should just have on by default? since the deleteOnExit seems pretty safe, since it'll wait for JVM to exit.

and we probably don't need a configuration flag around it then